pkg/proxy/proxy_test.go

```diff
@@ -263,12 +268,19 @@ func newTestProxy(t *testing.T) *fakeProxy {
 			clientTransport:       fakeRT,
 			noAuthClientTransport: fakeRT,
 			config:                new(Config),
+			hooks:                 hooks.New(),
 		},
 	}
 
+	auditor, err := audit.New(new(options.AuditOptions), "0.0.0.0:1234", new(server.SecureServingInfo))
+	if err != nil {
+		return nil, err
+	}
```

Comment:
nit: you can avoid returning an error here by calling `t.Fatalf` instead, which makes test code easier to read/write